metrics/lib/features/dashboard/presentation/model/project_metrics_data.dart

```diff
@@ -36,7 +34,6 @@ class ProjectMetricsData {
     this.numberOfBuilds,
```

Comment:
Maybe we should still have buildNumberMetrics but remove numberOfBuilds? Notice description:

/// [numberOfBuilds] is the number of builds the [buildNumberMetrics] are based on.

it still refers to it and I guess it will be more clear if we refer it as metric. What do you think?